src/operator/pooling.cu

```diff
@@ -15,7 +15,17 @@ namespace op {
 template<>
 Operator *CreateOp<gpu>(PoolingParam param) {
 #if MXNET_USE_CUDNN == 1
-  return new CuDNNPoolingOp(param);
+  switch (param.pool_type) {
+    case pool_enum::kMaxPooling:
+      return new CuDNNPoolingOp(param);
+    case pool_enum::kAvgPooling:
+      return new CuDNNPoolingOp(param);
+    case pool_enum::kSumPooling:
+      return new PoolingOp<gpu, mshadow::red::sum>(param);
```

Comment:
a LOG(WARNING) is added. Should I also change cudnnpooling? There is a with-case checking pooling_enum too.